### PR TITLE
Fig methods

### PR DESCRIPTION
**Pull request by aditya-sivakumar**
Hey Dr. Meyer,

We wrote up a couple of paragraphs for methods and added the figure captions in the manuscript files. Please let us know what to add, cut, expand on, etc!

Aditya

### CODE REVIEW

**Comment by aditya-sivakumar** [comment]
Vedant will also update GSEA portions soon

File: manuscript/06.methods.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 ## Methods
 
 All analysis was implemented in Python v3.8 and can be found at <https://github.com/meyer-lab/tfac-ccle>.
+
+### Data Import & Normalization
+We imported protein characterization data from the LINCS dataset, provided by Drs. Joe Gray, Laura Heiser, and James Korkola. The data came in three separate files, so they were concatenated together into one protein dataset.
+	
+For the gene expression dataset, we imported Oregon Health and Science University data which gave us a variety of biological markers’ data. For the genetic proponent, we used the RNA sequence data given by the OHSU data.
+
+Protein and gene expression data were not normalized after importation since it provided no significant difference in tensor factorization outcome.
```
Comment (aarmey) [inline manuscript/06.methods.md:10]: We didn't z-score the genes? If not we should do this.

Comment (aarmey) [reply manuscript/06.methods.md:10]: The protein data should also be normalized. I thought we did something there.

File: manuscript/06.methods.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+
+### GSEA 
+We tested for enrichment along the ARCHS4 gene list, which contains gene expression measurements from a breadth of human and mouse experiments: ARCHS4 was selected for our analysis since it is a comprehensive list of commonly tested RNA expression genes.5
```
Comment (aarmey) [inline manuscript/06.methods.md:13]: What are the numbers at the end of these sentences?

File: manuscript/06.methods.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+
+The ES of OHSU genes are determined by taking the maximum deviation of a running sum for each gene down the list from zero; this running sum increments by the magnitude of correlation between an observed phenotype for a given gene.4
+
+In addition to looking at ES, we want to identify genes that exhibit a low false discovery rate (FDR) in our decomposition, since it indicates a higher chance that the gene set is uniquely enriched for a given treatment and that it is not a false positive.
```
Comment (aarmey) [inline manuscript/06.methods.md:17]: What is a low FDR? Define.

**Review by aarmey — CHANGES_REQUESTED** [review]

File: manuscript/06.methods.md (hunk 2 above)
Comment (linnetc1721) [reply manuscript/06.methods.md]: That's on me, I left them there in the Google Doc to remind myself about the references, but forgot to remove them from the text. Just removed them from the manuscript!

File: manuscript/06.methods.md (hunk 3 above)
Comment (linnetc1721) [reply manuscript/06.methods.md]: Updated!

**Comment by aarmey** [comment]
@aditya-sivakumar let me know if this is ready to review again.

**Comment by aditya-sivakumar** [comment]
> @aditya-sivakumar let me know if this is ready to review again.
I will check this out right now and then Shashank and I should finish up the pylint stuff soon. I was waiting on him to get some things merged I believe!

**Comment by aditya-sivakumar** [comment]
Ok yes, this should be all set for a new review I believe! There is one part after talking about normalization where Linnet and I will provide more details of how Vedant is specifically normalizing the data after we merge with all the new changes in master and he gets back to us.

**Review by aarmey — APPROVED** [review]
Good to merge.

**Comment by aditya-sivakumar** [comment]
"The base branch restricts merging to authorized users" :(

**Comment by aarmey** [comment]
Try again.

**Pull request merged**